src/main/java/Main.java

```diff
@@ -10,7 +12,39 @@ public static void main(String[] args) {
     }
 
     void run(Scanner scanner) {
-        // usupełnij metodę
+        String fileName = "countries.csv";
+        try {
+            Map<String, Country> countriesMap = readFileAndGetCountriesMap(fileName);
+            System.out.println("Podaj kod kraju, o którym chcesz zobaczyć informacje:");
+            String codeFromUser = scanner.nextLine();
+            checkOrContainsCountry(countriesMap, codeFromUser);
+        } catch (FileNotFoundException e) {
+            System.out.println("Brak pliku " + fileName + ".");
+        }
+    }
+
+    private static void checkOrContainsCountry(Map<String, Country> countriesMap, String codeFromUser) {
```

Comment:
Nazwa check sugeruje, że metoda jest jakimś predykatem i zwróci true/false. Jeżeli metoda ostatecznie coś wyświetla, to raczej używałbym czasownika print, np. `printCountryInfo()`, czy `printCountryData()`